en/tutorials/video_streaming_wifi_broadcast.md

```diff
@@ -81,7 +81,9 @@ Be aware of [ground loop](https://en.wikipedia.org/wiki/Ground_loop_%28electrici
 ### Using a Linux Laptop as GCS (Harder than using a RasPi)
 1. On **ground** Linux development computer:
    ```
-   sudo apt install libpcap-dev libsodium-dev python3-all python3-twisted
+   sudo apt install python3-all libpcap-dev libsodium-dev python3-pip python3-pyroute2 \
+            python3-future python3-twisted python3-serial iw virtualenv \
```

Comment:
do you really need all these? virtualenv for example is recommended if you are using a virtual env, but none of the other things here require it or mention it - it is purely optional based on developer setup.

Ditto all the rest - the only reason I ever install libpcap is for wireshark - why is it needed here? Is future needed?

I know there is no particular harm in installing more dependencies, but it does annoy people if you ask for things that are not needed. If they are all essential then great - but we might split out those that are likely to already be installed into their own line just to make that clear.

Reply:
All dependencies are needed. I've added full list because some users continuously open bug reports when unable to build project with missing dependencies.
libpcap is core dependency - it used for packet injection.
virtual env is needed to setup `stdeb` (python deb package packer) which is not exists in debian/ubuntu.

Reply:
Thanks. Makes sense.